pandora-common/src/space/space.ts

```diff
@@ -61,9 +61,17 @@ export const SpaceInviteSchema = z.object({
 	characterId: CharacterIdSchema.optional(),
 	/** The time when the invite expires */
 	expires: z.number().int().optional(),
+	/** Type of the invite */
+	type: z.enum(['joinMe', 'spaceBound']),
+	/** Creator of the invite */
+	createdBy: z.object({
+		accountId: AccountIdSchema,
+		characterId: CharacterIdSchema,
+		characterName: z.string(),
+	}),
```

Comment:
These changes cause existing spaces with existing invites to fail to load. The invites in `SpaceDataSchema` should be updated to use `ZodArrayWithInvalidDrop`.